.travis.yml

```diff
@@ -4,9 +4,13 @@ sudo: false
 matrix:
   include:
     - python: 2.7
-      env: PANDAS_VERSION=0.18.1 NUMPY_VERSION=1.11.1 SCIPY_VERSION=0.17.1 LIBGFORTRAN_VERSION=3.0
+      env: PANDAS_VERSION=0.18.1
+    - python: 2.7
+      env: PANDAS_VERSION=0.22.0
+    - python: 3.5
+      env: PANDAS_VERSION=0.18.1
     - python: 3.5
-      env: PANDAS_VERSION=0.19.2 NUMPY_VERSION=1.12.1 SCIPY_VERSION=0.19.0 LIBGFORTRAN_VERSION=3.0
+      env: PANDAS_VERSION=0.22.0
```

Comment:
FWIW I think we could also spell this out as the product of the two vars:
```yaml
python:
  - 2.7
  - 3.5
env:
  - PANDAS_VERSION=0.18.1
  - PANDAS_VERSION=0.22.0
```

Reply:
that's much cleaner; fixed that